src/open_samus_returns_rando/specific_patches/cosmetic_patches.py

```diff
@@ -48,3 +50,37 @@ def music_shuffle(editor: PatcherEditor, configuration: dict) -> None:
         original_track = f"sounds/streams/music/{original}.bcwav"
         new_track = temp_dict[f"sounds/streams/music/{new}.bcwav"]
         editor.replace_asset(original_track, new_track)
+
+
+def volume_patches(editor: PatcherEditor, configuration: dict) -> None:
+    music = configuration["volume_adjustments"]["music"]
+    environment_sfx = configuration["volume_adjustments"]["environment_sfx"]
+    # If there are no custom values, quit
+    if music == 1 and environment_sfx == 1:
```

Comment:
I'd do this check slightly differently:
have `_adjust_music_volume()` and `_adjust_sfx_volume()` as seperate functios
Then `if music != 1: _adjust_music_volume(); if environment_sfx != 1: _adjust_sfx_volume();`
That way, you also skip out on unnecessay modification if you do `music = 1, environment_sfx = 0`